content/contributions/latest/propose-content.md

```diff
@@ -0,0 +1,5 @@
+---
+---
+# Propose content
```

Comment:
@jurgen-publiq can you add some info what the purpose of these new pages is? They seem unfinished and have no actual info right now.

Reply:
Maybe something better suited for a separate PR?